src/ui/bar-chart/BarChart.scss

```diff
@@ -0,0 +1,48 @@
+@import "styles/mixins";
+
+.barchart {
+    position: relative;
+    height: auto;
+    background-color: $surface;
+    padding: 30px 30px 50px 30px;
+
+    .wrapper {
+        @include rowEnd;
+        padding: 0 40px;
+        margin-top: 40px;
+        background: repeating-linear-gradient(
+            to top,
+            rgba($primary, 0.5),
+            rgba($primary, 0.5) 1px,
+            transparent 1px,
+            transparent 39px
+        );
+    }
+
+    .header {
```

Comment:
header  and inside  & > some element withour class if have a class & > not needed

Reply:
done